### PR TITLE
Fix the Admin UI Tests

### PR DESCRIPTION
**Pull request by dvaldivia**
This fixes the admin.ts tests for the new menu
Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -76,9 +76,12 @@ test-permissions:
 	@(env bash $(PWD)/portal-ui/tests/scripts/permissions.sh)
 	@(docker stop minio)
 
+test-apply-permissions:
+	@(env bash $(PWD)/portal-ui/tests/scripts/initialize-env.sh)
+
 initialize-permissions:
 	@(docker run -d --name minio --rm -p 9000:9000 quay.io/minio/minio:latest server /data{1...4})
-	@(env bash $(PWD)/portal-ui/tests/scripts/initialize-env.sh)
+	test-apply-permissions
```
Comment (bexsoft) [inline Makefile:84]: This target is being detected as a command during execution, I think you'll need to create one more target that includes both `test-apply-permissions` & `initialize-permissions`

File: portal-ui/src/screens/Console/Menu/Menu.tsx [hunk 2]
```diff
@@ -290,27 +290,27 @@ const Menu = ({
...
+        //   icon: InspectMenuIcon,
+        //   to: IAM_PAGES.INSPECT,
+        // },
+        // {
+        //   name: "Profile",
+        //   id: "profile",
+        //   component: NavLink,
+        //   icon: ProfileMenuIcon,
+        //   to: IAM_PAGES.PROFILE,
+        // },
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Menu/Menu.tsx:313]: Please remove this as well as their imports:
```
src/screens/Console/Menu/Menu.tsx
  Line 61:3:  'InspectMenuIcon' is defined but never used   @typescript-eslint/no-unused-vars
  Line 62:3:  'ProfileMenuIcon' is defined but never used   @typescript-eslint/no-unused-vars
  Line 75:8:  'CallHomeMenuIcon' is defined but never used  @typescript-eslint/no-unused-vars
```

**Review by bexsoft — CHANGES_REQUESTED** [review]

**Comment by kaankabalak** [comment]
Hi @dvaldivia, I have fixed the 4 tests that were returning errors locally on the commit below, please feel free to cherry-pick it into your branch:
https://github.com/kaankabalak/console/commit/554d843122ca7e8a1adf2bce7389eb9b875a8e7f

**Review by cniackz — APPROVED** [review]

**Review by Alevsk — APPROVED** [review]

**Review by bexsoft — COMMENTED** [review]
LGTM

**Pull request merged**